sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectSummary.groovy

```diff
@@ -3,6 +3,7 @@ package life.qbic.portal.sampletracking.components.projectoverview
 import groovy.transform.EqualsAndHashCode
 import java.time.Instant
 import life.qbic.business.project.Project
+import java.util.Comparator
```

Comment:
why do you need it here?

Reply:
we don't, leftover from having the comparator in this file